README.md

```diff
@@ -2,21 +2,20 @@
 
 # Concepts in Computer Science: Algorithms
 
-
 ## Objectives
 
-By the end of this, developers should be able to:
+By the end of this, developers should be able to...
 
 -   Define "algorithm"
 -   Identify what Big-O time-complexity measures
 -   Give an example of a divide-and-conquer algorithm
 -   Predict the time-complexity of a given algorithm
 
-## Computer Science (0:05, 5 min)
+## Computer Science (5 minutes, 0:05)
 
 How many of us find the concept or mention of computer science intimidating?
 
-Many think of  computer science as a necessary prerequisite to do anything programming-related with a computer and *really know what you are doing*, or that knowing a certain amount of computer science is some kind of litmus test for *true programmers*. This attitude perhaps unnecessarily mystifies an already difficult subject and field of study, and at worst, makes the learning curve seem so steep that it is like a learning barrier.
+Many think of computer science as a necessary prerequisite to do anything programming-related with a computer and *really know what you are doing*, or that knowing a certain amount of computer science is some kind of litmus test for *true programmers*. This attitude perhaps unnecessarily mystifies an already difficult subject and field of study, and at worst, makes the learning curve seem so steep that it is like a learning barrier.
 
```

Comment:
"This attitude perhaps unnecessarily mystifies an already difficult subject and field of study and, at worst, ..."

move the after `study`